### PR TITLE
Pulling the wiki into the repo

### PR DESCRIPTION
**Pull request by vringar**
The wiki was horribly out of date since it was never checked out to disk and never updated alongside the code.
This PR aims to change that by moving all files that previously existed as part of wiki into a wiki folder.

### CODE REVIEW

**Comment by vringar** [comment]
@englehardt While the wiki is not perfect, I think getting it into the tree is more important than having it be perfects once it's in there. 
I hope the warning on wiki/Platform-Architecture.md is clear enough, but I'm willing to delete it if you want me to.

File: wiki/Available-Commands.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Available Commands
```
Comment (englehardt) [inline wiki/Available-Commands.md:1]: Aren't all of these commands currently documented in `Instruments.md`? We should drop this page until we have proper docs available.

File: wiki/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+## General Hints and Guidelines
+
+### Avoid failing tests for PRs caused by formatting/linting issues
+
+If you have `pre-commit` running you should pass all linting in the CI as the lints run before every commit.
+If you have chosen not to use pre-commit we recommend you run
+a combination of `isort` and `autopep8` to fix errors automatically in-place before you submit your PR:
+
+* `python -m isort -rc --skip venv .`
+* `python -m autopep8 -r --in-place --exclude venv .`
```
Comment (englehardt) [inline wiki/CONTRIBUTING.md:32]: Should this now be updated to black?

File: wiki/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+### Updating schema docs
+
+In the rare instance that you need to create schema docs
+(after updating or adding files to `schemas` folder), run `npm install`
+from OpenWPM top level. Then run `npm run render_schema_docs`. This will update the
+`docs/schemas` folder. You may want to clean out the `docs/schemas` folder before doing this
+incase files have been renamed.
+
+## Troubleshooting
```
Comment (englehardt) [inline wiki/CONTRIBUTING.md:123]: All of these issues may hit a person just trying to install and run OpenWPM? They should be moved back into the main README. I think we can merge them with the other troubleshooting section.

File: wiki/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+in the test directory to run all tests:
+
+    $ conda activate openwpm
+    $ cd test
+    $ py.test -vv
+
+See the [pytest docs](https://docs.pytest.org/en/latest/) for more information on selecting
+specific tests and various pytest options.
+
+### Mac OSX
```
Comment (englehardt) [inline wiki/CONTRIBUTING.md:103]: This info is useful just for people trying to run OpenWPM on Mac OSX. We should move back into the main README.

File: wiki/Instrumentation-Schema-Documentation.md [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+* [http_redirects](#http_redirects)
+* [javascript](#javascript)
+* [javascript_cookies](#javascript_cookies)
+* [navigations](#navigations)
+* [callstacks](#callstacks)
+* [incomplete_visits](#incomplete_visits)
+
+This is an overview of all tables currently existing in OpenWPM. Over time we want to add
+a description for all fields and tables here.
+If you have any questions on any field, please file an issue or ask in the OpenWPM riot channel and we'll
```
Comment (englehardt) [inline wiki/Instrumentation-Schema-Documentation.md:15]: Add a link to the riot channel here.

File: wiki/Platform-Architecture.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
+# !!Warning!!
```
Comment (englehardt) [inline wiki/Platform-Architecture.md:1]: Should we just delete this?

**Review by englehardt — CHANGES_REQUESTED** [review]
I've left some comments on the changes, two additional notes:
1. It seems strange to separate the instrumentation configuration description for the rest of the configuration descriptions. I think we should merge the current `Instrumentation.md` file with the config documentation under `Browser and Platform Configuration` and `Browser Profile Support`. Those three sections cover all the config questions a person may have.
2. Let's place these under the docs folder, rather than create a new `wiki` folder.

**Review by englehardt — APPROVED** [review]
Thanks Stefan, great work! I checked through the major changes since the last review and they look great. This is a big step forward in our documentation :)

**Pull request merged**